src/arrays/Arrays.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace arrays;
+
+class Arrays implements ArraysInterface
+{
+    /**
+     * The $input variable contains an array of digits
+     * Return an array which will contain the same digits but repetitive by its value
+     * without changing the order.
+     * Example: [1,3,2] => [1,3,3,3,2,2]
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param array $input
+     * @return array
+     */
+    public function repeatArrayValues(array $input): array
+    {
+        $array = [];
+        foreach($input as $value){
+            $tempArray = array_fill(0,$value,$value);
+            $array = array_merge($array,$tempArray);
+        }
+        return $array;
+    }
+
+    /**
+     * The $input variable contains an array of digits
+     * Return the lowest unique value or 0 if there is no unique values or array is empty.
+     * Example: [1, 2, 3, 2, 1, 5, 6] => 3
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param array $input
+     * @return int
+     */
+    public function getUniqueValue(array $input): int
+    {
+        if(count($input)==0){
+            return 0;
+        }
+        $array = [];
+        foreach (array_count_values($input) as $key=>$value){
```

Comment:
Please change it to 
`foreach (array_count_values($input) as $value => $valueCount) {`